src/less/apps.less

```diff
@@ -95,6 +95,58 @@ img.icon-enricher {
 /* ----------------------------------------- */
 
 .sfrpg {
+
+    // Bubbles
+    .bubble {
+        margin: 0.5em 0.5em 1em 0.5em;
+        border: 2px solid #235683;
+        background-color:rgba(128, 128, 128, 0.2);
+        border-radius: 13px;
+
+        .bubble-info {
+            padding: 0.25em 1em 0.5em 1em;
+
+            .bubble {
+                margin: 0.5em 0.5em 0.5em 0.5em;
+                border: 2px solid #444444;
+                background-color:rgba(128, 128, 128, 0.2);
+                border-radius: 13px;
+    
+                .bubble-info {
+                    padding: 0.25em 1em 0.25em 1em;
+                }
+    
+                .bubble-header {
+                    background-color: #444444;
+                    color: white;
+                    border-style: none;
+                    border-top-left-radius: 9px;
+                    border-top-right-radius: 9px;
+                    margin: 0 0 0 0;
+                    padding: 0 0 3px 0;
+                    text-align: center;
+                    .exo2();
+                    font-size: var(--font-size-16);
+                    font-weight: normal;
+                }
+            }
+        }
+
+        .bubble-header {
+            background-color: #235683;
+            color: white;
+            border-style: none;
+            border-top-left-radius: 9px;
+            border-top-right-radius: 9px;
+            margin: 0 0 0 0;
+            padding: 0 0 3px 0;
+            text-align: center;
+            .exo2();
+            font-size: var(--font-size-24);
```

Comment:
I also think font size 16 or 18 would be nicer here, again for less dead space

Reply:
Sure, I've swapped it over to 18